src/applications/check-in/components/tests/AddressBlock.unit.spec.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+
+import AddressBlock from '../AddressBlock';
+
+describe('check-in', () => {
+  describe('AddressBlock', () => {
+    const address = {
+      address1: 'line 1',
+      address2: 'line 2',
+      address3: 'line 3',
+      city: 'city',
+      state: 'state',
+      zip: '00000',
+    };
+    it('Renders', () => {
```

Comment:
Do you a test where it only shows `address line 1`. Also, the test ids should be `kebab-casing` and shouldn't have spaces